docs/news.rst

```diff
@@ -3,6 +3,202 @@
 Release notes
 =============
 
+.. _release-2.4.0:
+
+Scrapy 2.4.0 (2020-10-??)
+-------------------------
+
+Highlights:
+
+*   Python 3.5 support has been dropped
+
+*   The ``file_path`` method of :ref:`media pipelines <topics-media-pipeline>`
+    can now access the source :ref:`item <topics-items>`
+
+    This allows you to set a download file path based on item data.
+
+*   You can now choose whether :ref:`feed exports <topics-feed-exports>`
```

Comment:
What do you thhink about explaining here what it means for CLI? It can become a very common option to use.